apps/look-and-feel-stories/src/components/LevelSelector/LevelSelector.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { LevelSelector } from "@axa-fr/canopee-react/client";
+import { Meta, StoryObj } from "@storybook/react";
+import { useState } from "react";
+
+const meta: Meta<typeof LevelSelector> = {
+  title: "Components/LevelSelector",
+  component: LevelSelector,
+};
+export default meta;
+
+export const Default: StoryObj<typeof LevelSelector> = {
+  render: ({ ...args }) => {
+    const [value, setValue] = useState(args.value || 0);
+    return (
+      <div>
+        <LevelSelector {...args} value={value} onChange={setValue} />
+      </div>
+    );
+  },
+  args: {
+    steps: 3,
+    title: "Titre du niveau actif",
+    description:
+      "Texte secondaire dynamique sur le niveau actif avec un contenu plus ou moins long selon le besoin",
+    value: 0,
+    onChange: (value: number) => {
+      console.log("Selected level:", value);
+    },
+  },
```

Comment:
Plutôt que de mettre un console.log dans ton onChange (qui est d'ailleurs pour le moment surchargé par ton setValue), tu peux faire cela afin de voir dans l'onglet actions de storybook les appels de fonctions détaillées.

```suggestion
  args: {
    steps: 3,
    title: "Titre du niveau actif",
    description:
      "Texte secondaire dynamique sur le niveau actif avec un contenu plus ou moins long selon le besoin",
    value: 0,
  },
  argTypes: {
    onClick: { action: "onClick" },
  }
```